tests/foreman/api/test_docker.py

```diff
@@ -725,41 +810,118 @@ def test_promote_multiple_docker_repo_content_view(self):
+        ).create()
+        content_view.repository = [repo]
+        content_view = content_view.update(['repository'])
+        self.assertEqual(
+            [repo.id], [repo_.id for repo_ in content_view.repository])
+
+        content_view.publish()
+        cvv = content_view.read().version[0].read()
+
+        comp_content_view = entities.ContentView(
+            composite=True,
+            organization=self.org,
+        ).create()
+        comp_content_view.component = [cvv]
+        comp_content_view = comp_content_view.update(['component'])
+        self.assertEqual(cvv.id, comp_content_view.component[0].id)
+
+        comp_content_view.publish()
+        comp_cvv = comp_content_view.read().version[0]
+        self.assertEqual(len(comp_cvv.read().environment), 1)
+
+        comp_cvv.promote({u'environment_id': lce.id})
+        self.assertEqual(
+            len(comp_content_view.read().version[0].read().environment), 2)
 
-    @stubbed()
     @run_only_on('sat')
     def test_promote_multiple_docker_repo_composite_content_view(self):
-        """@Test: Add Docker-type repository to composite content view and
-        publish it. Then promote it to the multiple available
-        lifecycle-environments.
+        """@Test: Add Docker-type repository to content view and publish it.
+        Then add that content view to composite one. Publish and promote that
+        composite content view to the multiple available lifecycle-environments
 
         @Assert: Docker-type repository is promoted to content view found in
         the specific lifecycle-environments.
 
         @Feature: Docker
 
-        @Status: Manual
-
         """
+        product = entities.Product(organization=self.org).create()
+        repo = _create_repository(product)
+        content_view = entities.ContentView(
+            composite=False,
+            organization=self.org,
+        ).create()
+        content_view.repository = [repo]
+        content_view = content_view.update(['repository'])
+        self.assertEqual(
+            [repo.id], [repo_.id for repo_ in content_view.repository])
+
+        content_view.publish()
+        cvv = content_view.read().version[0].read()
+
+        comp_content_view = entities.ContentView(
+            composite=True,
+            organization=self.org,
+        ).create()
+        comp_content_view.component = [cvv]
+        comp_content_view = comp_content_view.update(['component'])
+        self.assertEqual(cvv.id, comp_content_view.component[0].id)
+
+        comp_content_view.publish()
+        comp_cvv = comp_content_view.read().version[0]
+        self.assertEqual(len(comp_cvv.read().environment), 1)
+
+        for i in range(1, randint(2, 5)):
+            lce = entities.LifecycleEnvironment(organization=self.org).create()
+            comp_cvv.promote({u'environment_id': lce.id})
+            self.assertEqual(
+                len(comp_content_view.read().version[0].read().environment),
```

Comment:
Can you replace `comp_content_view.read().version[0].read().environment` with `comp_cvv.read().environment`?